.lintstagedrc.json

```diff
@@ -1,5 +1,6 @@
 {
   "*.{ts,js}": ["pnpm lint:fix", "prettier --write"],
```

Comment:
Is this redundant now with the addition of `"*": ["pnpm lint:fix", "prettier --write"]`?